ansible/inventories/staging

```diff
@@ -1,2 +1,2 @@
 [staging]
-thegreenwebfoundation.org tgwf_stage="staging" tgwf_domain_name=staging-newadmin project_root="/var/www/{{ tgwf_domain_name }}.thegreenwebfoundation.org"
\ No newline at end of file
+thegreenwebfoundation.org tgwf_stage="staging" tgwf_domain_name=staging-newadmin project_root="/var/www/{{ tgwf_domain_name }}.thegreenwebfoundation.org" gunicorn_port=9003
```

Comment:
Do we have separate databases for staging and production, right now?

Reply:
Not yet. I'm absolutely in favour of us having one.

A few years back I used this on a project, and I figure it's been in use, so might be worth trying out:

https://github.com/18F/rdbms-subsetter